app/presenters/hyrax/displays_image.rb

```diff
@@ -0,0 +1,39 @@
+require 'iiif_manifest'
+
+module Hyrax
+  # This gets mixed into FileSetPresenter in order to create
+  # a canvas on a IIIF manifest
+  module DisplaysImage
+    extend ActiveSupport::Concern
+
+    # Creates a display image only where FileSet is an image.
+    #
+    # @return [IIIFManifest::DisplayImage] the display image required by the manifest builder.
+    def display_image
+      return nil unless ::FileSet.exists?(id) && solr_document.image?
+      # TODO: this is slow, find a better way (perhaps index iiif url):
+      original_file = ::FileSet.find(id).original_file
+
+      # TODO: We want url to be like https://libimages1.princeton.edu/loris/plum/0c%2F48%2F3j%2F48%2F8-intermediate_file.jp2/full/!200,200/0/default.jpg
```

Comment:
Why is this a TODO?

Reply:
that appears to be a hangover from the implementation in Hyku ... it should go.

Reply:
Because you made it one in Hyku, I think. :smiley:

Reply:
https://github.com/samvera-labs/hyku/commit/829a73a88f4c2f52af14a1e2ed4a9972e92bad27#diff-71e459f9baa1f414befed67e735a7206R11